staff/durack/subpages/salinity/index.html

```diff
@@ -0,0 +1,150 @@
+                    *.tif file suitable for print.</p>
+            </div> <!--id=content-->
+
+            <div id="content" class="index">
+                <img style="padding: 5px; border: thin solid white;" src="png/120406_CTD_Retrieval.png" alt="CSIRO"
+                    align="left">
+                <p>Photo 3. Lined with bottles triggered at different levels of
+                    the ocean, this conductivity, temperature and depth profiler bearing a
+                    suite of sampling bottles is a mainstay of oceanography. It can be
+                    deployed to depths of 6,000 metres to study changes in ocean temperature
+                    and salinity. (CSIRO) - Clicking <a href="tif/120410_CTD_Retrieval.tif"><strong>HERE</strong></a>
+                    will download a high-resolution *.tif file suitable for print.</p>
+            </div> <!--id=content-->
+
+            <div id="content" class="index">
+                <img style="padding: 5px; border: thin solid white;" src="png/120405_SouthernOceanInSituMonitoring.png"
+                    alt="CSIRO" align="left">
+                <p>Photo 4. Monitoring of the Southern Ocean using arrays of
+                    anchored and drifting instruments reveals freshening of deep waters
+                    around Antarctica. Some of the extra melting of ice around the edge of
+                    Antarctica is flowing into the sea and getting carried down to the deep
+                    ocean by ocean currents. (CSIRO: Steve Rintoul) - Clicking <a
+                        href="tif/120410_SouthernOceanInSituMonitoring.tif"><strong>HERE</strong></a> will download a
+                    high-resolution *.tif file suitable for print.</p>
+            </div> <!--id=content-->
+
+            <div id="content" class="index">
+                <img style="padding: 5px; border: thin solid white;" src="png/120405_Argo_SouthernOceanDeployment.png"
+                    alt="CSIRO: Alicia Navidad" align="left">
+                <p>Photo 5. Akin to having a fleet of miniature research vessels,
+                    the global flotilla of more than 3,000 robotic profiling floats provides
+                    crucial information on upper layers of the world's ocean currents.
+                    (CSIRO: Alicia Navidad) - Clicking <a
+                        href="tif/120410_Argo_SouthernOceanDeployment.tif"><strong>HERE</strong></a> will download a
+                    high-resolution *.tif file suitable for print.</p>
+            </div> <!--id=content-->
+
+            <div id="content" class="index">
+                <img style="padding: 5px; border: thin solid white;" src="png/120412_GlobalSalinityMP4.png" alt="CSIRO"
+                    align="left">
+                <p>Video 1. Selected footage of oceanographic videos for use in broadcast (CSIRO) - Clicking <a
+                        href="http://mbf.cc/xFF6f"><strong>HERE (783Mb)</strong></a> will download a high-definition
```

Comment:
@durack1 These mbf.cc links are not working.  Do you have these files?  We could host them on our server instead.

Reply:
@mauzey1 thanks for checking, I just took a scour and don't seem to have downloaded these files to my personal archive - I wonder if these are in a backup somewhere, doubtful if they are not close to the directory

Reply:
@mauzey1 just found a [youtube video](https://www.youtube.com/watch?v=odG83ruW33Q) which should be a reasonable switchout - I'll tweak this PR and ping you when it's in place this afternoon